pkg/gardenlet/apis/config/types.go

```diff
@@ -200,6 +200,10 @@ type SeedControllerConfiguration struct {
 	// LeaseResyncSeconds defines how often (in seconds) the seed lease is renewed.
 	// Default: 2s
 	LeaseResyncSeconds *int32
+	// LeaseResyncGracePeriod is the amount of missed lease resyncs before the health status
```

Comment:
I know it was this way before, but to be precise, it's not really a period since it's not related to time, is it? Maybe we should better call it `LeaseResyncMissThreshold` or something similar to make it more precise?

Reply:
Renamed `LeaseResyncGracePeriod ` to `LeaseResyncMissThreshold `.